### PR TITLE
Add links to datasets

### PR DESCRIPTION
**Pull request by mbonsma**
Fixes issue #10

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -296,8 +296,11 @@ <h2 id="setup">Setup</h2>
 
 <p>
   To participate in a Software Carpentry workshop, you will need
-  access to the software described below. In addition, you will
-  need an up-to-date web browser.
+  access to the software described below as well as an up-to-date web browser. 
+  You will also need access to the 
+  <a href = "https://swcarpentry.github.io//shell-novice/shell-novice-data.zip">shell dataset</a>
+  and the <a href = "https://github.com/swcarpentry/r-novice-gapminder/blob/gh-pages/data/gapminder-FiveYearData.csv">
+  gapminder dataset</a>.
```
Comment (QuLogic) [inline index.html]: Last time we didn't really need the gapminder dataset because it was installed as an R package; is it needed this time?

Comment (mbonsma) [inline index.html]: Good point. I'll take it off.

File: index.html [hunk 2]
```diff
@@ -296,8 +296,11 @@ <h2 id="setup">Setup</h2>
 
 <p>
   To participate in a Software Carpentry workshop, you will need
-  access to the software described below. In addition, you will
-  need an up-to-date web browser.
+  access to the software described below as well as an up-to-date web browser. 
+  You will also need access to the 
+  <a href = "https://swcarpentry.github.io//shell-novice/shell-novice-data.zip">shell dataset</a>
+  and the <a href = "https://github.com/swcarpentry/r-novice-gapminder/blob/gh-pages/data/gapminder-FiveYearData.csv">
```
Comment (lwjohnst86) [inline index.html]: You might want to use this link: https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/data/gapminder-FiveYearData.csv

This link goes directly to the dataset, which they can just copy and paste into a text editor/spreadsheet.

Other option is to use `wget https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/data/gapminder-FiveYearData.csv` which downloads it even more directly... it is uses bash :)

File: index.html (hunk 1 above)
Comment (lwjohnst86) [inline index.html]: Maybe include a bit about installing the gapminder package? I can't recall, did you do that during the workshop last time?

Comment (QuLogic) [inline index.html]: Yes, Erin did it last time; I think it's part of the lesson plan, too?

**Comment by QuLogic** [comment]
Speaking of, I just noticed our reference link points to R's inflammation lesson; are you doing that one, or the gapminder lesson?

**Comment by mbonsma** [comment]
Gapminder fo shizzle. I'll change that too.

**Pull request merged**